src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const perDay = 40;
+  const longDay = 7;
+  const shortDay = 3;
+  const discLongDay = 50;
+  const discShortDay = 20;
+
+  const total = days * perDay;
+
+  if (days >= longDay) {
+    return total - discLongDay;
+  }
+
+  if (days >= shortDay && days < longDay) {
+    return total - discShortDay;
+  }
+
+  return total;
```

Comment:
The function `calculateRentalCost` is well-named and describes its purpose. However, the constants `longDay` and `shortDay` could be more descriptive. Consider renaming them to `LONG_TERM_RENTAL_DAYS` and `SHORT_TERM_RENTAL_DAYS` respectively, to clearly indicate their purpose related to rental duration. Similarly, `discLongDay` and `discShortDay` could be renamed to `LONG_TERM_DISCOUNT` and `SHORT_TERM_DISCOUNT` for clarity.